blueliquiddesigns/gravity-forms-pdf-extended
# Changes to meet all WordPress.org Plugin Guidelines

## Description

1. Additional disclosure in readme file about 3rd party services
2. Escape variables on echo, even if escaped at an earlier point, or the variable was created without user-generated content 
3. Fix WordPress-Extra PHPCS lint issues
4. Security hardening
5. Readability

## Testing instructions
Install on a fresh WordPress installation and run through every piece of functionality to verify functionality. 

## Screenshots 

## Checklist:

## Additional Comments
